### PR TITLE
Add description of infrastructure in Pulumi

### PR DESCRIPTION
**Pull request by arnenator**
# What is this PR about?
This PR contains change to a new workflow and a description of our infrastructure in code. The nice thing about this is that every time we have a change in the infrastructure code and it is pushed to main, Pulumi takes care of the changes.

Also, I have created an integration between Pulumi and Ansible such that every time we push something new to main then Ansible is going to take the servers and push Configuration Management changes to the servers. This also includes deploying the new application.

### CODE REVIEW

**Review by MalinNordqvist — APPROVED** [review]
LGTM

File: Minitwit.Provision/Infrastructure/DigitalOcean/DODatabaseCluster.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        Large => "db-s-4vcpu-8gb",
+        _ => throw new NotSupportedException($"The size {size} is not supported for Database Nodes in DigitalOcean Databases")
+    };
+
+    public void CreateDatabase(string name)
+    {
+        new DatabaseDb(name, new()
+        {
+            Name = name,
+            ClusterId = _cluster.Id, // TODO: Add db name in the new args
```
Comment (Pinkvinus) [inline Minitwit.Provision/Infrastructure/DigitalOcean/DODatabaseCluster.cs:93]: TODO?

Comment (arnenator) [reply Minitwit.Provision/Infrastructure/DigitalOcean/DODatabaseCluster.cs]: Solved

**Review by MalinNordqvist — APPROVED** [review]
LGTM

**Pull request merged**